src/attributes.jl

```diff
@@ -850,6 +850,23 @@ struct TimeLimitSec <: AbstractOptimizerAttribute end
 
 attribute_value_type(::TimeLimitSec) = Union{Nothing,Float64}
 
+"""
+    ObjectiveLimit()
+
+An optimizer attribute for setting a limit on the objective value.
+
+The provided limit must be a `Union{Float64,Nothing}`.
```

Comment:
It's not Float64. For a solver using BigFloat, it can be BigFloat. Let's just say it's Real

Reply:
done